vital/algo/algorithm.h

```diff
@@ -53,95 +49,85 @@ class VITAL_ALGO_EXPORT algorithm
   /// Get this algorithm's \link kwiver::vital::config_block configuration
   /// block \endlink
 
```

Comment:
It looks like the tool isn't combining comments of different styles. Besides that it would be really good to do that, I'm somewhat concerned it could lead to problems later, such as the brief and detailed descriptions getting merged.
```suggestion
///
```

Reply:
I agree it is not combining comments of different styles, but that it because they were not combined in the original. I don't see how this could combine brief and detailed descriptions unless they were immediately adjacent (i.e. no intervening blank line) in the original.

Reply:
Ah, I see, I didn't realize you were using something found, rather than something you'd written. Maybe @mleotta would also like to comment, but given that a) I'm not 100% confident the changes as-is don't introduce a problem, and b) this is a systemic issue that should be easier to fix *before* changing the comment style, I think it's probably good to try to fix it now.

Reply:
To clarify: the situation right now is that a `///` comment followed by a `/**` comment should basically always be repaired as per my prior comment. Once we're looking at all `///` comments which may or may not have a blank line, I think it's going to be harder to tell whether we should just remove the blank line, or replace it with an empty `///`, or do something else...

Reply:
Makes sense. I'll look into a fix of the kind you mentioned a few comments up.

Reply:
As noted, I have high confidence that converting `(///[^\n]+\n)\n?/[*][*]` to `\1///\n/**` is always correct; I'm pretty sure Doxugen treats the change of comment style as a paragraph break. I'm less confident what it does with separate `///` blocks, or if those already exist for other reasons. In any case, it's an issue that needs to be fixed, and shouldn't add significantly more "noise".